src/utilities/_hypre_utilities.h

```diff
@@ -1308,6 +1308,11 @@ typedef struct
    HYPRE_MemoryLocation   memory_location;
    HYPRE_ExecutionPolicy  default_exec_policy;
    HYPRE_ExecutionPolicy  struct_exec_policy;
+   /* the device buffers needed to do MPI communication for struct comm */
```

Comment:
What is the reason for moving these variables to `hypre_handle`? Thanks!

Reply:
Moving these out of the hypre_DeviceData structure allows me to macro out hypre_DeviceData when using device openmp (i.e. all the hypre_DeviceData stuff is just behing HYPRE_USING_GPU and not DEVICE_OPENMP).

Reply:
Thanks for clarifying!